tests/conftest.py

```diff
@@ -1,35 +1,35 @@
-import os
-
 import pytest
 import vcr
 from click.testing import CliRunner
 
 from tim.opensearch import configure_opensearch_client
 
+EXIT_CODES = {
+    "success": 0,
+    "error": 1,
+    "invalid_command": 2,
+}
 my_vcr = vcr.VCR(
     cassette_library_dir="tests/fixtures/cassettes",
     filter_headers=["authorization"],
 )
 
 
 @pytest.fixture(autouse=True)
-def test_env():
-    os.environ = {
-        "AWS_ACCESS_KEY_ID": "test",
-        "AWS_SECRET_ACCESS_KEY": "test",
-        "AWS_SESSION_TOKEN": "test",
-        "TIMDEX_OPENSEARCH_ENDPOINT": "localhost",
-        "SENTRY_DSN": None,
-        "WORKSPACE": "test",
-    }
-    yield
-
-
-@pytest.fixture()
+def _test_env(monkeypatch):
+    monkeypatch.setenv("SENTRY_DSN", "None")
+    monkeypatch.setenv("WORKSPACE", "test")
+    monkeypatch.setenv("AWS_ACCESS_KEY_ID", "test")
+    monkeypatch.setenv("AWS_SECRET_ACCESS_KEY", "test")
+    monkeypatch.setenv("AWS_SESSION_TOKEN", "test")
+    monkeypatch.setenv("TIMDEX_OPENSEARCH_ENDPOINT", "localhost")
```

Comment:
Good call on the `monkeypatch` update; this has proven much easier to work with in other repos.